src/main/java/io/gingersnapproject/cdc/cache/hotrod/HotRodCacheBackend.java

```diff
@@ -15,25 +17,36 @@ public class HotRodCacheBackend implements CacheBackend {
    final RemoteCache<String, String> remoteCache;
    final JsonTranslator<?> keyTranslator;
    final JsonTranslator<?> valueTranslator;
+   private final NotificationManager eventing;
+   private final BiConsumer<Object, Throwable> eventBiConsumer;
 
-   public HotRodCacheBackend(URI uri, RemoteCache<String, String> remoteCache, JsonTranslator<?> keyTranslator,
-         JsonTranslator<?> valueTranslator) {
+   public HotRodCacheBackend(URI uri, RemoteCache<String, String> remoteCache,
+                             JsonTranslator<?> keyTranslator, JsonTranslator<?> valueTranslator,
+                             NotificationManager eventing) {
       this.uriUsed = uri;
       this.remoteCache = remoteCache.withDataFormat(DataFormat.builder()
             .keyType(MediaType.TEXT_PLAIN).valueType(MediaType.TEXT_PLAIN).build());
       this.keyTranslator = keyTranslator;
       this.valueTranslator = valueTranslator;
+      this.eventing = eventing;
+      this.eventBiConsumer = (ignore, t) -> {
+         if (t != null) {
+            this.eventing.backendFailedEvent(uriUsed, t);
+         }
+      };
    }
 
    @Override
    public CompletionStage<Void> remove(Json json) {
       return remoteCache.removeAsync(keyTranslator.apply(json).toString())
+            .whenComplete(eventBiConsumer)
```

Comment:
The returned stage will still have the exception if it failed exceptionally. Do we want to now ignore that? If so we should use `handle` below instead.

Reply:
I think it's better to return the stage with the exception, so the engine is aware the batch was not processed.

Reply:
Okay, I didn't know if the error handler in the consumer was sufficient to stop the backend or not.